refs/schemas/priv/dictionary/other/en.other.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Those serve for the aswer. Like when you ask whether the system has any data on you. The system can answer "YES".

I am changeing the term family from `OTHER` to `BOOLEAN` in order to avoid confusion with the Term `OTHER` that is used to indicate a particular subcategory of something

@Clementinev -> please apply the change in examples if needed